client/dashboard/sites/overview/index.tsx

```diff
@@ -116,12 +117,16 @@ function SiteOverview( {
 						<BackupCard site={ site } />
 					</VStack>
 					<VStack className="site-overview-cards" spacing={ spacing }>
-						<OverviewCard
-							title={ __( 'Performance' ) }
-							icon={ chartBar }
-							heading="TBA"
-							description="TBA"
-						/>
+						{ site.is_a4a_dev_site ? (
```

Comment:
The preview link feature is available if the site is Coming Soon / Unlaunched. Should we check here? Or is that automatically true if the site if `.is_a4a_dev_site`

Reply:
We only intend to show this for A4A development sites as a replacement for the Performance card. So for Coming Soon / Unlaunched we want to show Performance instead. Good question though, I'm not sure why we want to hide the Performance card for A4A dev sites. cc: @matt-west 

<img width="232" height="87" alt="SCR-20250717-jrnw" src="https://github.com/user-attachments/assets/a710f7a0-3cef-4c33-b74a-b887e78e8282" />

Reply:
That makes sense 🙂

Reply:
Please disregard; I just tested and as soon as I launch the site, it's no longer `.is_a4a_dev_site` 😄 (you can also double check)

Reply:
> I'm not sure why we want to hide the Performance card for A4A dev sites.

Just to add some extra context on this, performance reports aren’t available until the site has been made public so we wouldn’t have anything to show for A4A dev sites. (Which cannot be public under a dev license.)

We replace this card for unlaunched .com sites too. They see a migration CTA instead.

<img width="3456" height="2234" alt="image" src="https://github.com/user-attachments/assets/948849b7-a9f4-4d09-9f26-a17fd44c0e3d" />

Reply:
Makes sense, thanks for the context 👍